openwisp_controller/connection/connectors/airos/snmp.py

```diff
@@ -0,0 +1,5 @@
+from ..snmp import Snmp as BaseSnmp
+
+
+class Snmp(BaseSnmp):
+    pass
```

Comment:
I think there should be a similar file for openwrt

Reply:
Acc to this, if we create `connectors/openwrt/snmp.py` as  :
```py
from ..snmp import Snmp as BaseSnmp


class OpenWRTSnmp(BaseSnmp):
    pass

```

And Changed `connectors/airos/snmp.py` as :
```py
from ..snmp import Snmp as BaseSnmp


class AirOsSnmp(BaseSnmp):
    pass
```
Then settings would look like this : 
```py
 ('openwisp_controller.connection.connectors.openwrt.snmp.OpenWRTSnmp', 'OpenWRT SNMP'),
    (
        'openwisp_controller.connection.connectors.airos.snmp.AirOsSnmp',
        'Ubiquiti AirOS SNMP',
    ),
```
But according to your above review https://github.com/openwisp/openwisp-controller/pull/496#discussion_r874087524 if we want to achieve below settings 
```py
openwisp_controller.connection.connectors.snmp.OpenWrtSnmp
openwisp_controller.connection.connectors.snmp.AirOsSnmp
```
then both `OpenWrtSnmp` & `AirOsSnmp` class must be inside `connection/connector/snmp.py`
```py
#  connection/connector/snmp.py

class Snmp(object):
....
....

class OpenWRTSnmp(Snmp):
    pass

class AirOsSnmp(Snmp):
    pass
```
I'm confused which should we do, or I'm missing something

Reply:
go with the first option, thanks!